examples/plot_glm_retinal_ganglia.py

```diff
@@ -0,0 +1,289 @@
+# -*- coding: utf-8 -*-
+"""
+=================================================================
+GLM for Spike Trains Prediction in Primate Retinal Ganglion Cells
+=================================================================
+
+* Original tutorial adapted from Johnathan Pillow, Princeton University
+* Dataset provided by E.J. Chichilnisky, Stanford University
+* The dataset is granted by the original authors for educational use only
+* Please contact ``pillow@princeton.edu`` if using beyond its purposes
+
+First of all, we would like to thank Professor Johnathan Pillow
+and Professor E.J. Chichilnisky for sharing their tutorial dataset.
+The original MATLAB and Python tutorial can be found from
+https://github.com/pillowlab/GLMspiketraintutorial.
+
+These data were collected by Valerie Uzzell in the lab of
+E.J. Chichilnisky at the Salk Institute. For full information see
+Uzzell et al. (J Neurophys 04) [1]_, or Pillow et al. (J Neurosci 2005) [2]_.
+
+In this tutorial, we will demonstrate how to fit linear GLM and
+linear-nonlinear GLM (a.k.a Poisson GLM) to predict the spike counts
+recorded from primate retinal ganglion cells.
+The dataset contains spike responses from 2 ON and 2 OFF parasol
+retinal ganglion cells (RGCs) in primate retina, stimulated with
+full-field `binary white noise`. Two experiment performed consisted of a
+long (20-minute) binary stochastic (non-repeating) stimulus
+which can be used for computing the spike-triggered average
+(or characterizing some other model of the response).
+
+References
+----------
+.. [1] Uzzell, V. J., and E. J. Chichilnisky. "Precision of
+   spike trains in primate retinal ganglion cells."
+   Journal of Neurophysiology 92.2 (2004)
+.. [2] Pillow, Jonathan W., et al. "Prediction and decoding of
+   retinal ganglion cell responses with a probabilistic
+   spiking model." Journal of Neuroscience 25.47 (2005)
+
+"""
+
+# Authors: Titipat Achakulvisut <my.titipat@gmail.com>
```

Comment:
Sorry if I wasn't clear, you can add the original author names here. Only I don't think you need to specify that you edited it because the example would have changed substantially.

Reply:
Sounds good! I will change it to the original author here.